src/config/config_documentation/pod_cidr.adoc

```diff
@@ -0,0 +1,5 @@
+This setting controls the pod address range that the agent reports to Kubernetes.
```

Comment:
This should have something to this effect prominently:

Note: There should almost never be a reason to use this setting, this is mostly here for very special circumstances. Do not touch it unless you really know what you're doing.

Reply:
I've added a warning to this effect.